landing-page/scripts/auth_api.py

```
@@ -756,4 +757,24 @@ def logout(x_refresh_token: str = Header(None, alias="X-Refresh-Token"), user=De
+    mei_api.py's create_edit_session (re-verifies an existing one)."""
+    try:
+        cur.execute(
+            "SELECT alignment_json FROM text_alignments WHERE image_name=%s AND project_id=%s"
+            " ORDER BY created_at DESC LIMIT 1",
+            (image_name, project_id),
+        )
+        row = cur.fetchone()
+        if row and row[0]:
+            return json.loads(row[0])
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)